python/pyspark/sql/functions.py

```diff
@@ -189,15 +210,15 @@ def _():
 }
 
 for _name, _doc in _functions.items():
-    globals()[_name] = since(1.3)(_create_function(_name, _doc))
+    globals()[_name] = since(1.3)(ignore_unicode_prefix(_create_function(_name, _doc)))
```

Comment:
Yea, I didn't mean to not use but avoid applying all other docs (more specifically the ones in `_functions`).  This basically replaces u prefix via regex and rather a workaround IMHO. So, I think we should avoid applying this to other doctests not using unicode.